ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -66,8 +66,6 @@ if ver hwcmp PX4FMU_V2
 then
 	# External I2C bus
 	hmc5883 -C -T -X start
-
-	# External I2C bus
```

Comment:
Should we leave this as is to avoid the rebase when FMUv1 comes out?

Reply:
Yes I suppose so.